gradle-plugin/plugin/src/main/kotlin/com/emergetools/android/gradle/tasks/snapshots/transform/FindCustomPreviewClassVisitor.kt

```diff
@@ -30,6 +30,17 @@ class FindCustomPreviewClassVisitor(
   override fun visitAnnotation(descriptor: String?, visible: Boolean): AnnotationVisitor? {
     if (descriptor == EMERGE_APP_STORE_SNAPSHOT) {
       currentAnnotation.isAppStoreSnapshot = true
+    } else if (descriptor == EMERGE_SNAPSHOT_CONFIG) {
+      return object : AnnotationVisitor(Opcodes.ASM9) {
+        override fun visit(name: String?, value: Any?) {
+          if (name == "precision") {
+            currentAnnotation.precision = value as? Float ?: 1.0f
```

Comment:
We handle this on the backend but not sure if we should flag values outside of the 0.0 to 1.0 range?

Reply:
Yeah I can throw